SA3/qa_web/models.py

```diff
@@ -56,6 +56,9 @@ class Questions(Post):
 
     def __str__(self):
         return self.title
+
+    def get_answer_queryset(self):
```

Comment:
This method is not tested though it is quite simple so it should be a quick and easy fix

Reply:
Added test for it.